challenge-4.js

```diff
@@ -0,0 +1,32 @@
+function maxMin(arr) {
+  if (!isArrValid(arr)) return "bad request!";
```

Comment:
Nice idea for checking the condition inside another function. This is something that you've done in almost all of your challenges and it is GOOD. Excellent.